newsfragments/145.bugfix

```diff
@@ -0,0 +1 @@
+Remove --basedir option from all subcommands (use --node-directory instead)
```

Comment:
I like Twisted's newsfragment style guide: https://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles

Though I notice now it doesn't cover things like CLI interfaces.  It seems like there is a logical way to extend it to this, though.  Can we adopt such a convention for magic-folder?

Reply:
Adopting some kind of guidelines would be good.

For this, I didn't really know whether to call it a bugfix or a feature (or even "incompat"?) and I guess from the guidelines it should be like a "now" sentence?

"--node-directory is now the only way to specify the Tahoe node"

or

"Sub-commands no longer accept the --basedir option; use --node-directory instead"

Reply:
Maybe "The --node-directory option is now required and is the only way to specify the Tahoe directory"?

Reply:
The middle option there looks best to me but the others look good as well.